Source/WebDriver/socket/HTTPParser.cpp

```diff
@@ -106,7 +106,7 @@ HTTPParser::Process HTTPParser::abortProcess(const char* message)
     m_phase = Phase::Error;
 
     if (!m_buffer.isEmpty())
```

Comment:
I don’t think this preflight makes things faster than the check `shrink` already does.